java/runtime/src/main/java/io/ray/runtime/RayDevRuntime.java

```diff
@@ -118,4 +127,12 @@ private AsyncContext(TaskSpec task) {
       this.task = task;
     }
   }
+
+  private static void updateSessionDir(RayConfig rayConfig) {
+    SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd_hh-mm-ss-ms");
+    Date date = new Date();
+    String sessionDir =
+        String.format("/tmp/ray/local_mode_session_%s_%d", format.format(date), SystemUtil.pid());
```

Comment:
Maybe we use `local_mode` as infix instead of prefix? like `/tmp/ray/session_local_mode_{time}`

Reply:
Good suggesstion.